federatedscope/core/fed_runner.py

```diff
@@ -166,6 +166,9 @@ def _setup_server(self, resource_info=None, client_resource_info=None):
         if self.cfg.vertical.use:
             from federatedscope.vertical_fl.utils import wrap_vertical_server
             server = wrap_vertical_server(server, self.cfg)
+        if self.cfg.fedswa.use:
```

Comment:
This implementation implies that, in the future, we'd better refactor the workers so that we would not need to make another worker class just for adding such a trick.